packages/abi-gen/test-cli/fixtures/contracts/AbiGenDummy.sol

```diff
@@ -88,4 +90,26 @@ contract AbiGenDummy
         return ecrecover(prefixedHash, v, r, s);
     }
 
+    event Event(uint8 param);
```

Comment:
Let's rename this to `AnEvent`. It's a bit confusing to have it identically named to the keyword.

Reply:
Added this to the checklist.